scripts/zones/Promyvion-Vahzl/mobs/Propagator.lua

```diff
@@ -5,6 +5,10 @@
 require("scripts/globals/missions")
 -----------------------------------
 
+function onMobSpawn(mob)
```

Comment:
Propagator might need to kill babies on death like Progenerator is.

Reply:
Good call. Fixed.